src/StaticFAQ/FAQArticleDetail.js

```diff
@@ -11,10 +11,22 @@ import { LanguageContext } from '../context/Language';
 import type { FAQArticleDetailQuery } from './__generated__/FAQArticleDetailQuery.graphql';
 
 const queryFAQArticleDetail = graphql`
-  query FAQArticleDetailQuery($id: ID!, $language: Language) {
+  query FAQArticleDetailQuery(
+    $id: ID!
+    $language: Language
+    $category_id: ID!
+  ) {
     FAQArticle(id: $id, language: $language) {
       ...FAQArticleDetailContent_article
     }
+    FAQCategory(id: $category_id) {
+      id
+      title
+      ancestors {
+        id
+        ...Breadcrumbs_breadcrumbs
```

Comment:
Here the current category should be included as well. As we've discussed, I had not implemented it yet, but should it be implemented now? Do you have a suggestion?